core/modules/widgets/navigator.js

```diff
@@ -564,7 +564,7 @@ NavigatorWidget.prototype.handlePerformImportEvent = function(event) {
 	// Navigate to the $:/Import tiddler
 	this.addToHistory([event.param]);
 	// Trigger an autosave
-	$tw.rootWidget.dispatchEvent({type: "tm-auto-save-wiki"});
+	$tw.rootWidget.dispatchEvent({type: "tm-auto-save-wiki", param: this.wiki.getTiddlerText("$:/config/SaveWikiButton/Template")});
```

Comment:
It seems awkward to have to add this parameter each time we invoke auto-save-wiki. Can we avoid it?

Reply:
You are right. It feels a bit hacky!

Reply:
The problem I see is, that the tiddler which causes the update is: `$:/config/SaveWikiButton/Template`. So we need to work with the active wiki-store.

Reply:
There are only 2 files, where `.dispatchEvent({type: "tm-auto-save-wiki", ...` is called:

 - navigator.js
 - plugin-maker.js